templates/index

```diff
@@ -293,6 +293,7 @@ paloalto_panos_show_interface_logical.template, .*, paloalto_panos, sh[[ow]] int
 paloalto_panos_show_counter_global.template, .*, paloalto_panos, sh[[ow]] coun[[ter]] glo[[bal]]
 paloalto_panos_show_system_info.template, .*, paloalto_panos, sh[[ow]] sys[[tem]] in[[fo]]
 paloalto_panos_show_jobs_all.template, .*, paloalto_panos, sh[[ow]] jo[[bs]] all
+paloalto_panos_show_mac_all.template, .*, paloalto_panos, sh[[ow]] mac all
```

Comment:
This is in conflict since I merged the other palo alto PR. Mind adding the show arp command on L295 above the mac command?

Reply:
fixed, thx!